app/src/stores/address.ts

```diff
@@ -0,0 +1,21 @@
+import { defineStore } from 'pinia'
+
+export const useOwnerAddressStore = defineStore({
+  id: 'ownerAddress',
+  state: () => ({
+    ownerAddress: localStorage.getItem('ownerAddress') || '' // Load owner address from localStorage
+  }),
+  actions: {
+    setOwnerAddress(address: string) {
+      this.ownerAddress = address
+      localStorage.setItem('ownerAddress', address) // Save owner address to localStorage
+    },
+    getOwnerAddress() {
+      return this.ownerAddress
+    },
```

Comment:
we don't have to create getters for it because we already create `ownerAddress` in state,

Reply:
ah yes. got it! thanks for pointing that out